nuget/Microsoft.Windows.CsWinRT.Authoring.targets

```diff
@@ -16,6 +16,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <CsWinRTDetectDependentAuthoringWinMDs Condition="'$(CsWinRTDetectDependentAuthoringWinMDs)'==''">true</CsWinRTDetectDependentAuthoringWinMDs>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <CsWinRTAuthoring_Platform Condition="'$(Platform)'=='AnyCPU'">x86</CsWinRTAuthoring_Platform>

```

Comment:
Are we relying on x86 can probably be loaded in most scenarios for anycpu?

Reply:
I assume this is because C++/Reunion apps can't be run with AnyCPU? Is this making x86 the default then and not x64?

Reply:
I have always seen AnyCPU / x86 / Win32 used interchangeably, which is why I did this. We have to pick a platform here because we don't make a WinRT.Host.dll for AnyCPU. We are assuming that someone who builds for AnyCPU will be fine using the x86 version of WinRT.Host.dll